### DISCUSSION TITLE
Deploy Elasticsearch and Kibana in Amazon Elastic Kubernetes Service (Amazon EKS)

### DISCUSSION THREAD

**Discussion by giscus** [appscode/community#58] [Blog]
# Deploy Elasticsearch and Kibana in Amazon Elastic Kubernetes Service (Amazon EKS)

AppsCode's latest news and announcements, tips and tricks for our cloud-native ecosystem and products such as KubeDB, Stash, Voyager, ByteBuilders, KubeVault, and many more.

https://byte.builders/blog/post/deploy-elasticsearch-and-kibana-in-aws/

**Comment by jamontesg** [comment]
Nice Tutorial, Do you have  a version using  ingress ?
